plugin/templates.vim

```diff
@@ -366,8 +366,8 @@ function <SID>TExpandVars()
 	if executable('licensee') && g:templates_use_licensee
         let l:projectpath = shellescape(expand("%:p:h"))
         if executable('git') && g:templates_detect_git
-            silent "!git rev-parse --is-inside-work-tree > /dev/null"
-            if v:shell_error == 0
+            let l:isgitrepo = matchstr(system("git rev-parse --is-inside-work-tree"), "true")
+            if l:isgitrepo == "true"
```

Comment:
Using `==` may do either case-sensitive or case-insensitive depending on the value of the `ignorecase` option, and I think it would be better to avoid depending on that.

Could you please change this comparison to use the `==#` operator to make sure that case-sensitive comparison is always used?